src/main/scala/firrtl/Emitter.scala

```diff
@@ -1078,6 +1092,16 @@ class VerilogEmitter extends SeqTransform with Emitter {
         emit(Seq("`endif // SYNTHESIS"))
       }
 
+      if (formals.keys.nonEmpty) {
+        emit(Seq("`ifdef FORMAL"))
```

Comment:
We should not ifdef these unless there is a specific reason we must.

Reply:
Fixed